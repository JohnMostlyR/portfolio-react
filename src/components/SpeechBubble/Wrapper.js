import styled from 'styled-components';
import {em} from 'polished';
import mq from '../../styles/templates/mediaQueries';
import typography from '../../styles/templates/typography';

import Bubble from './Bubble';

const Wrapper = styled.div`
  margin-bottom: 9vh;
  padding: 1vh 1vw;
  width: 100%;
  ${typography.doublePica};

  ${mq.xl`max-width: ${em('1008px')};`}

  ${Bubble} p {
    &:not(:first-child) {
      margin-top: 1.5rem;
    }
  }
`;

export default Wrapper;
