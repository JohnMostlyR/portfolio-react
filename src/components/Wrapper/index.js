import styled from 'styled-components';

const Wrapper = styled.div`
  display: flex;
  flex-flow: column;
  height: 100%;
  z-index: 10;
`;

export default Wrapper;
