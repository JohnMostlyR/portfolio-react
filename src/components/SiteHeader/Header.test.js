import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Wrapper from './Wrapper';

describe('Wrapper', () => {
  it('Should render and match the snapshot', () => {
    const tree = renderer.create(<Wrapper/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});