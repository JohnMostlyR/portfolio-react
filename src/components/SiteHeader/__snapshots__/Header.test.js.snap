// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper Should render and match the snapshot 1`] = `
.c0 {
  border-bottom: .5vh solid #575756;
}

.c0:after {
  content: "" !important;
  display: block !important;
  clear: both !important;
}

<div
  className="c0"
/>
`;
